doc/nrf/known_issues.rst

```diff
@@ -90,15 +91,26 @@ Sending data before connecting to nRF Cloud
   The :ref:`asset_tracker` sample does not wait for connection to nRF Cloud before trying to send data.
   This causes the sample to crash if the user toggles one of the switches before the board is connected to the cloud.
 
+.. rst-class:: v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0 v0-4-0 v0-3-0
+
+Missing support for FOTA on nRF Cloud
+  The :ref:`asset_tracker` sample does not support the nRFCloud FOTA_v2 protocol.
```

Comment:
Two different spellings for nRF Cloud here - which should really be nRF Connect for Cloud (but I don't think we've updated that everywhere).

Reply:
I'll leave this consistency update for a separate task, as there are some more occurrences outside of known issues: https://projecttools.nordicsemi.no/jira/browse/NCSDK-7785